README.md

```diff
@@ -215,6 +214,7 @@ The *forEach* function performs the following steps:
 1. ReturnIfAbrupt(*observerNext*).
 1. Let *promiseCapability* be NewPromiseCapability(%Promise%).
 1. ReturnIfAbrupt(*promiseCapability*).
+1. If IsCallable(*callbackfn*) is **false**, Call(promiseCapability.[[Reject]], TypeError exception).
```

Comment:
isn't 

> 1. Return *promiseCapability*.[[Promise]].

implied by being in a promise constructor +  throwing ? Or am i confusing JS side-affects with spec lang requirements?